packages/d1/src/api.ts

```diff
@@ -30,32 +30,38 @@ interface ResponseMeta {
 interface SuccessResponse {
   results: any;
   duration: number;
-  lastRowId: number | null;
-  changes: number | null;
   success: true;
   served_by: string;
   meta: ResponseMeta | null;
+  // These are deprecated in place of `meta`
+  lastRowId: number | null;
+  changes: number | null;
```

Comment:
I think we can remove these now. They've never had data in them, and now they never will (only inside `.meta`)

Reply:
Yeah if we were keeping these it'd be more correct to hardcode the type to `null` tho